lib/run_loop/device_agent/client.rb

```diff
@@ -793,8 +782,13 @@ def fail(*several_variants)
         raise exception_type, message
       end
 
+=begin
+PRIVATE
+=end
       private
 
+      attr_reader :http_client
+
```

Comment:
@krukow 

> http_client vs. retriable_http_client

I prefer `http_client` because the DeviceAgent::Client does not care that it is the `HTTP::Retriable::Client` - any HTTP client will do as long as it responds to:

```
* delete
* post
* get
* reset_all!
* timeout
* retries
* interval
```